string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,9 @@
+/* exported isVowel */
+function isVowel(char) {
+  if (char === 'a' || char === 'e' || char === 'i' || char === 'o' || char === 'u') {
```

Comment:
We haven't really covered these, but you can also use something called a `switch` statement here. Additional reading [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch) if you're interested

```
function isVowel(character) {
  switch (character.toLowerCase()) {
    case 'a':
    case 'e':
    case 'i':
    case 'o':
    case 'u':
      return true;
    default:
      return false;
  }
}
```